bio/falco/wrapper.py

```
@@ -0,0 +1,54 @@
+import re
+from tempfile import TemporaryDirectory
+from snakemake.shell import shell
+
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+if len(snakemake.input) > 1:
+    raise IOError("Got multiple input files, I don't know how to process them!")
+
+# Output directory
+output_dir_user = snakemake.output.get("dir", "")
```

Ruff (0.14.7)

Undefined name `snakemake`

(F821)